pkg/nsx/services/subnet/builder.go

```diff
@@ -41,7 +41,7 @@ func (service *SubnetService) buildSubnetSetName(subnetset *v1alpha1.SubnetSet,
 }
 
 func (service *SubnetService) buildSubnet(obj client.Object, tags []model.Tag) (*model.VpcSubnet, error) {
-	tags = util.AppendTags(service.buildBasicTags(obj), tags)
+	tags = append(service.buildBasicTags(obj), tags...)
```

Comment:
In the base functions of util:
https://github.com/vmware-tanzu/nsx-operator/blob/vpc_dev/pkg/util/utils.go#L40
we add common.TagScopeNamespace, common.TagScopeNamespaceUID, however, these two tags should not be included in vm subnet, we need to revise this to make sure all resource except vm subnet to add these two.

Reply:
So, you mean there is a bug in util.AppendTags for subnetset, is it?
if yes, can we fix it by changing util.AppendTags funcs to handle subnet/subnetset in proper way? I think util.AppendTags is common func provided for resources.

Reply:
refer https://github.com/vmware-tanzu/nsx-operator/pull/336